pkg/manager/internal.go

```diff
@@ -185,8 +186,9 @@ type controllerManager struct {
 	// is closed because it will be nil.
 	shutdownCtx context.Context
 
-	internalCtx    context.Context
-	internalCancel context.CancelFunc
+	internalCtx            context.Context
+	internalCancel         context.CancelFunc
+	internalProceduresStop chan struct{}
```

Comment:
Could you add a comment mentioning that this is only required for the `StopChannelInto`?

Reply:
Done ptal